easybuild/easyblocks/g/go.py

```diff
@@ -61,7 +61,7 @@ def install_step(self):
 
         # $GOROOT_FINAL only specifies the location of the final installation, which gets baked into the binaries
         # the installation itself is *not* done by the all.bash script, that needs to be done manually
-        cmd = "GOROOT_FINAL=%s ./all.bash" % self.installdir
+        cmd = "GOROOT_BOOTSTRAP=%s GOROOT_FINAL=%s ./all.bash" % (os.environ.get('EBROOTGO'),self.installdir)
```

Comment:
maybe it's also good to add a comment why `$GOROOT_BOOTSTRAP` must be defined now too?

should we make this specific to Go 1.5+?

```python
if LooseVersion(self.version) >= LooseVersion('1.5'):
   ...
```